server/src/types/environment.d.ts

```diff
@@ -0,0 +1,12 @@
+export {};
+
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+        ACCESS_TOKEN_SECRET: string;
+        REFRESH_TOKEN_SECRET: string;
+        ACCESS_TOKEN_EXPIRATION_IN_MINUTES: string;
```

Comment:
In `.env`, increase the expiration time significantly (1 minute and 5 minutes is super low, I believe you did so in order to validate what you did)